tests/specs/MultisigImportanceTransferModule.spec.js

```diff
@@ -0,0 +1,188 @@
+import WalletFixture from '../data/wallet';
+import AccountDataFixture from '../data/accountData';
+
+describe('Multisig Importance transfer module tests', function() {
+    let $controller, $rootScope, Wallet, DataBridge, $filter, Nodes, SuperNodeProgram, $timeout;
+
+    beforeEach(angular.mock.module('app'));
+
+    beforeEach(angular.mock.inject(function(_$filter_, _$controller_, _$rootScope_, _$timeout_, _Wallet_, _DataBridge_, _Nodes_, _SuperNodeProgram_) {
+        $controller = _$controller_;
+        $rootScope = _$rootScope_;
+        $timeout = _$timeout_;
+        Wallet = _Wallet_;
+        DataBridge = _DataBridge_;
+        $filter = _$filter_;
+        Nodes = _Nodes_;
+        SuperNodeProgram = _SuperNodeProgram_;
+    }));
+
+    function createDummyWalletContextTestnet(Wallet) {
+        Wallet.use(WalletFixture.testnetWallet);
+        Nodes.setDefault();
+        DataBridge.accountData = AccountDataFixture.testnetAccountData;
+    }
+
+    function createDummyWalletContextMainnet(Wallet) {
+        Wallet.use(WalletFixture.mainnetWallet);
+        Nodes.setDefault();
+        DataBridge.accountData = AccountDataFixture.mainnetAccountData;
+    }
+
+    describe('setNodes', () => {
```

Comment:
please refactor as these tests are identical to those in previous file - this is generally an indication of a code smell / refactoring opportunity

Reply:
yeah, I will work on this refactor.